static/src/js/components/CollectionSelector/__tests__/CollectionSelector.test.jsx

```diff
@@ -229,7 +233,7 @@ describe('CollectionSelector', () => {
               params: {
                 options: { shortName: { pattern: true } },
                 shortName: 'C*',
-                limit: 20
+                limit: 100
```

Comment:
why 100 and not bigger?

Reply:
Just makes a very large list, 100 was 5x more than what was previously used, so I figured I was pushing the UI/UX concept of that box as it was.

Reply:
The real solution is to allow better search criteria, right now it is just short name.

Reply:
what is the cost in supporting 2000?

Reply:
I think it is more of a usability issue vs cost.  smaller list, easier to find results, etc.

Reply:
answered in person.